### PR TITLE
Added URL Scheme for Scriptable.

### PR DESCRIPTION
**Pull request by frenchesco**

### CODE REVIEW

File: data/apps/scriptable.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+  "info": {
+    "identifier": "scriptable",
+    "name": "Scriptable",
+    "appstoreUrl": "https://itunes.apple.com/au/app/scriptable/id1405459188?mt=8"
```
Comment (pietropizzi) [inline data/apps/scriptable.json]: Is there no documentation link? Where do you have the docs from?

File: data/apps/scriptable.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  "actions": [
+    {
+      "name": "Open Scriptable",
+      "description": "Opens Scriptable.",
+      "url": "scriptable://"
+    },
+    {
+      "name": "Add Script",
+      "description": "Add a new script.",
+      "url": "scriptable:///add"
```
Comment (pietropizzi) [inline data/apps/scriptable.json]: There is on `/` too many in here.

Also I don't understand this action. How is it that an `add` action does not have any parameters? Will this just create a completely empty script? This is where the link to some official docs would be great for more details.

File: data/apps/scriptable.json [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    },
+    {
+      "name": "Open Script",
+      "description": "Add a new script.",
+      "url": "scriptable:///open",
```
Comment (pietropizzi) [inline data/apps/scriptable.json]: One `/` too many

File: data/apps/scriptable.json [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+              "description": "false - Script settings will not be opened."
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "name": "Run Script",
+      "description": "Run a script.",
+      "url": "scriptable:///run",
```
Comment (pietropizzi) [inline data/apps/scriptable.json]: And one more `/`

File: data/apps/scriptable.json (hunk 1 above)
Comment (frenchesco) [reply data/apps/scriptable.json]: The documentation is documented within the app so I couldn't link to it. I searched Google for other documentation, but didn't find any. Here's a copy/paste of the documentation from within the app:

### URLScheme
**Manages URL schemes for Scriptable.**

Use URL schemes to launch the app and perform an action, such as running a script.
The app conforms to the scriptable:// URL scheme. The following documents actions that can be performed using the URL scheme.

**Adding a script**
To add a new script, you should use the following URL scheme:

`scriptable:///add`

**Opening a script**
To open an existing script, you should use the following URL scheme:

`scriptable:///open?scriptName=Example`

The scriptName query parameter is the name of the script to open. scriptName must be URL encoded. You may optionally add the query parameter openSettings with a value of true to automatically open the script settings.

**Running a script**
To run an existing script, you should use the following URL scheme:

`scriptable:///run?scriptName=Example`

The scriptName query parameter is the name of the script to run. scriptName must be URL encoded.

**Screenshot**
image

Comment (pietropizzi) [reply data/apps/scriptable.json]: Cool thanks for the explanation! Once we have the docs in here, I'll try and reach out to the app maker and ask if they want to provide the docs somewhere outside the app too.

If you could just fix the `///` to `//` then I can merge and deploy this

File: data/apps/scriptable.json (hunk 2 above)
Comment (pietropizzi) [reply data/apps/scriptable.json]: Wow thats super weird

**Pull request merged**